src/test/java/mil/dds/anet/test/integration/config/AnetTestConfiguration.java

```diff
@@ -13,7 +13,7 @@
   private static final HashMap<String, Object> config;
   private static final ObjectMapper yamlMapper = new ObjectMapper(new YAMLFactory());
   private static final File configFile =
-      new File("src/integration-test/resources/anet-integrationtest.yml");
+      new File("src/test/java/mil/dds/anet/test/integration/resources/anet-integrationtest.yml");
```

Comment:
Shouldn't this file be in `src/test/resources` and loaded through the resource loader?